PyPDF2/_utils.py

```diff
@@ -241,17 +241,13 @@ def paeth_predictor(left: int, up: int, up_left: int) -> int:
         return up_left
 
 
+def deprecate(msg: str, stacklevel: int = 3) -> None:
+    warnings.warn(msg, PendingDeprecationWarning, stacklevel=stacklevel)
+
+
 def deprecate_with_replacement(old_name: str, new_name: str) -> None:
-    warnings.warn(
-        DEPR_MSG.format(old_name, new_name),
-        PendingDeprecationWarning,
-        stacklevel=2,
-    )
+    deprecate(DEPR_MSG.format(old_name, new_name), 4)
```

Comment:
Nitpic: I try to use keyword arguments when I use literal parameters, e.g.:

```suggestion
    deprecate(DEPR_MSG.format(old_name, new_name), stacklevel=4)
```

I think this is easier to read.